public/index.html

```diff
@@ -79,6 +79,16 @@
             <img src="img/trashcan2.png"/>
           </span>
         </li>
+        <li :class="'legend_end legend_item legend_item_' + configuration.legendDisplay + ' legend_list_' + configuration.legendDisplay + '_bottom'">
+          <query-times
```

Comment:
> `' legend_list_' + configuration.legendDisplay + '_bottom'` 

This will generate a CSS class `legend_list_bottom_bottom`, which does not exist. I'm not sure if I like that.

Reply:
I don't like it, but we will fix that in a seperate PR. See also #170